portfolio/src/main/webapp/script.js

```diff
@@ -55,16 +55,21 @@ function loadSkills () {
  * Makes a request to /data and gets the response
  */
 async function fetchFromData() {
-  const response = await fetch('/data');
+  const selector = $("#num-comments")[0]; //get the select element
+  const selectedNum = selector.options[selector.selectedIndex].value; //number of comments user wants
```

Comment:
I think you can just do $("#num-comments").val() to get the value of the selected element.
https://api.jquery.com/val/

Reply:
oh my gosh I feel so dumb thank you!